sdcm/cluster_aws.py

```diff
@@ -306,9 +302,51 @@ def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
                 fi
             done
 
-            sudo ip r a $IPv6_CIDR dev eth0
+            sudo sh -c "cat <<EOT >> /etc/cloud/cloud.cfg.d/99-custom-networking.cfg
+            network:
+              version: 1
+              config:
+              - type: physical
+                name: eth0
+                subnets:
+                  - type: dhcp
+                  - type: dhcp6
+            EOT"
+
+            sudo sh -c "cat <<EOT >> /etc/rc.local
+            sh -c 'echo IPV6_AUTOCONF=yes >> /etc/sysconfig/network-scripts/ifcfg-eth0'
```

Comment:
Also, this is not what I wrote you to add...

I asked for this: 
/etc/sysconfig/network-scripts/init.ipv6-global
sudo ip r a $IPv6_CIDR dev eth0